link/uprobe.go

```diff
@@ -49,9 +49,13 @@ type Executable struct {
 // UprobeOptions defines additional parameters that will be used
 // when loading Uprobes.
 type UprobeOptions struct {
-	// Symbol offset. Must be provided in case of external symbols (shared libs).
+	// Symbol offset (absolute). Must be provided in case of external symbols (shared libs).
```

Comment:
this shouldn't be the offset from the start (so maybe `absolute` is not the best term). Maybe leave this comment as is and specify in the comment below that `RelativeOffset` is an additional offset relative to the calculated or provided offset?

Reply:
@mmat11 reverted with https://github.com/mertyildiran/ebpf/commit/ddbe83e447b992cedb652554f987d0a8c83e0f50